src/main/java/net/mcreator/ui/gradle/GradleConsole.java

```diff
@@ -286,7 +284,7 @@ public void exec(String command, @Nullable GradleTaskFinishedListener taskSpecif
 		StyleConstants.setFontSize(keyWord, 4);
 		pan.insertString("\n", keyWord);
 
-		append("Executing Gradle task: " + command, new Color(0xBBD9D0));
+		append(L10N.t("gradle.executing_task" + command), new Color(0xBBD9D0));

```

Comment:
Command name is not part of a translation here. Please fix it.
`L10N.t("gradle.executing_task" + command)`
<kbd>-></kbd>
`L10N.t("gradle.executing_task") + command`